rest-service/manager_rest/rest/resources_v3_1/manager_config.py

```diff
@@ -49,3 +49,25 @@ def _authorization_config():
             'permissions': cfy_config.authorization_permissions
         }
         return authorization
+
+
+class ManagerRestConfig(SecuredResource):
+    @rest_decorators.exceptions_handled
+    @authorize('manager_rest_config_get')
+    def get(self):
+        """
+        Get the Manager's REST configuration
+        :return: Manager's REST configuration data
+        """
+        request_args = request.args.to_dict(flat=False)
+        current_app.logger.info('Retrieving Manager REST config. filter: {0}'.
+                                format(request_args))
+        result = dict()
+        filter_by = request_args.get('filter', [])
+        for key in filter_by:
+            result[key] = getattr(config.instance, key)
```

Comment:
Need to have default values and/or handle exceptions when `key` is not an attribute that the config has.

Better yet, instead of making it a general rest call that gets all of the config, maybe we should only return the configurations that we absolutely must.

Reply:
getattr's default value is None.
The reason behind returning everything is for future uses, I didn't want to create this REST endpoint just for this feature to retrieve the database IP, that's why I added the filter.

Reply:
No it's not - you'll get an `AttributeError`.

Reply:
Right, my bad, I looked in the wrong place.
fixed